sense.py

```diff
@@ -38,6 +57,86 @@ def __call__(self, tensor):
         return proxy
 
 
+class RSTD(CSTransform): #Random Sampling Time Domain
+
+    def __init__(self, compression_factor, img_shp):
+ 
+        super().__init__(compression_factor, img_shp)
+        
+        
+        self.A = np.eye(self.N)
+        all_idx = np.arange(0, self.N, 1)
+        idx = self.rng.integers(0, self.N, (self.M,), endpoint=True)
+        dif = np.setdiff1d(all_idx, idx)
+        self.A[dif, :] = 0
```

Comment:
Select rows instead of zeroing out the rest

Reply:
Did this for RSTD